buildSrc/src/main/kotlin/addMavenCentralRepository.kt

```diff
@@ -0,0 +1,50 @@
+import com.google.common.base.CaseFormat
+import org.gradle.api.Project
+import org.gradle.api.publish.PublishingExtension
+import org.gradle.api.publish.maven.MavenPublication
+import org.gradle.api.publish.maven.tasks.PublishToMavenRepository
+import org.gradle.kotlin.dsl.withType
+import java.net.URI
+
+fun PublishingExtension.addMavenCentralRepository(project: Project, publication: MavenPublication) {
```

Comment:
Renamed the existing Bintray version and converted it to Maven Central.

It might make sense to look at a diff between the two while reviewing. (shame the rename obscures the changes on Github)